app/cinema/bar.py

```diff
@@ -0,0 +1,12 @@
+from app.people.customer import Customer
+
+
+class CinemaBar:
+    def sell_product(self, product: str, customer: str) -> None:
+
+        if isinstance(customer, Customer):
+            customer_name = customer.name
+        else:
+            customer_name = customer
+
+        print(f"Cinema bar sold {product} to {customer_name}.")
```

Comment:
```suggestion
class CinemaBar:
    @staticmethod
    def sell_product(product: str, customer: Customer) -> None:
        print(f"Cinema bar sold {product} to {customer.name}.")
```

<img width="750" height="422" alt="image" src="https://github.com/user-attachments/assets/92fff9b6-baae-4a7a-930f-f42e3b5d470b" />